lib/Tapable.js

```diff
@@ -128,19 +128,19 @@ Tapable.prototype.applyPluginsBailResult1 = function applyPluginsBailResult1(nam
 
 Tapable.prototype.applyPluginsAsyncSeries = Tapable.prototype.applyPluginsAsync = function applyPluginsAsyncSeries(name) {
 	var args = Array.prototype.slice.call(arguments, 1);
-	var callback = args.pop();
+	var callback = args[args.length - 1];
```

Comment:
Not really. Original codes remove the last element here, but push a new one after a while. Replacing the last element directly will be faster, without breaking changes. Please refer to the original implementation of `applyPluginsParallelBailResult`.